OctoKit/PullRequest.swift

```diff
@@ -233,12 +309,27 @@ enum PullRequestRouter: JSONPostRouter {
                 parameters["head"] = head
             }
 
+            return parameters
+        case let .patchPullRequest(_, _, _, _, title, body, state, base, mantainerCanModify):
+            var parameters = [
+                "title": title,
+                "state": state.rawValue, // FIXME: All is invalid here?
```

Comment:
Is this still valid? I saw you added a guard let earlier which is fine by me.

Reply:
Missed removing it, I'll push an update removing the fix